packages/core/src/js/feedback/FeedbackFormManager.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { Modal, StyleSheet,View } from 'react-native';
+
+import { FeedbackForm } from './FeedbackForm';
+
+class FeedbackFormManager {
+  private static _isVisible = false;
+  private static _setVisibility: (visible: boolean) => void;
+
+  public static initialize(setVisibility: (visible: boolean) => void): void {
+    this._setVisibility = setVisibility;
+  }
+
+  public static show(): void {
+    if (this._setVisibility) {
+      this._isVisible = true;
+      this._setVisibility(true);
+    }
+  }
+
+  public static hide(): void {
+    if (this._setVisibility) {
+      this._isVisible = false;
+      this._setVisibility(false);
+    }
+  }
+
+  public static isFormVisible(): boolean {
+    return this._isVisible;
+  }
+}
+
+interface FeedbackFormProviderProps {
+  children: React.ReactNode;
+}
+
+class FeedbackFormProvider extends React.Component<FeedbackFormProviderProps> {
+  public state = {
+    isVisible: false,
+  };
+
+  public constructor(props: FeedbackFormProviderProps) {
+    super(props);
+    FeedbackFormManager.initialize(this._setVisibilityFunction);
+  }
+
+  /**
+   * Renders the feedback form modal.
+   */
+  public render(): React.ReactNode {
+    const { isVisible } = this.state;
+
+    return (
+      <>
+        {this.props.children}
+        {isVisible && (
+          <Modal visible={isVisible} transparent animationType="slide">
+            <View style={styles.modalBackground}>
+              <FeedbackForm
+                onFormClose={this._handleClose}
+                onFormSubmitted={this._handleClose}
+              />
+            </View>
+          </Modal>
```

Comment:
Just wrapping this in `<View></View>` no styles nothing fixes the issue.

Reply:
This works as expected on both platforms :) Thank you Krystof 🙇 
Updated with 011fea7fafab764ac49e240a7c21cbffb541da97